app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -544,7 +535,7 @@ private void createPlayer(int buffer, boolean isInterlaced) {
             options.add("-v");
             options.add("--vout=android-opaque,android-display");
 
-            mLibVLC = new LibVLC(TvApp.getApplication(), options);
+            mLibVLC = new LibVLC(mActivity, options);
```

Comment:
was the `TvApp.getApplication()` just missed in a previous PR and removing it is okay, or was it left in for a reason?
Same for the toast on line 556

Reply:
Probably forgotten

Reply:
Yeah forgotten, I tend to be good with big picture stuff but miss little details like this :+1: 
Thanks for fixing that